test-data/unit/check-flags.test

```diff
@@ -515,3 +515,59 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
+[case testDisallowAnyExprUnannotatedFunction]
+# flags: --disallow-any=expr
+def g(s):
+    return s  # E: Expression has type "Any"
```

Comment:
@ddfisher I think Github may have hidden this comment. I think that's something worth including in documentation. 

What's your opinion on this?